drivers/serial/uart_nrfx_uart.c

```diff
@@ -14,7 +14,12 @@
 
 
 #ifdef CONFIG_UART_INTERRUPT_DRIVEN
-static uart_irq_callback_t m_irq_callback; /**< Callback function pointer */
+
+static uart_irq_callback_t irq_callback; /**< Callback function pointer */
+/* Help variable used by some functions to return correct driver status before
+ * first byte is shifted out from UART0. */
```

Comment:
Strictly speaking, this is not specific to UART"0"

Reply:
it is specific to UART0 because in all our chips we have either UART0 only or no UART at all, only UARTE.
For UARTE there is simpler warkaround.

Reply:
We have UART1 in Graviton, at least.
I know that the driver, now, uses UART0, but in theory, we could use UART1 as well. The comment could cover all. So "UART" is better, I think.